rakelib/form526.rake

```diff
@@ -118,6 +118,15 @@ namespace :form526 do
       end
     end
 
+    def clean_message(msg)
```

Comment:
Oh just noticed, where this came from. I prefer leaving this logic directly below the `messages.each do |msg|` iterator so it's clear what `msg` represents.

To avoid the max line length rubocop, you could write it like:

```ruby
message = if msg[1].present?
            "#{msg[0]}: #{msg[1].gsub(/GUID.*/, '')}"
          else
            msg[0]
          end
```

This is all probably a nitpick though 😬

Reply:
yeah, this is too awkward.  I'm going to clean this up another PR.